suite/tests/api/ir_riscv64.expect

```diff
@@ -221,7 +221,7 @@ c.xor  a5 -> fp
 c.sub  a5 -> fp
 test_integer_arith complete
 lui    0x2a -> a0
-auipc  0x2a -> a0
+jalr   a1 0x2a -> a0
```

Comment:
Looks like the instrs tested by test_instr_encoding_pc_related are not in the expected output at all?

Reply:
I understand. My question was: should we also add expected outputs for the instrs tested by test_instr_encoding_pc_related?